h2o-automl/src/main/java/ai/h2o/automl/AutoMLBuildSpec.java

```diff
@@ -49,8 +49,8 @@ public AutoMLBuildControl() {
     public float max_after_balance_size = 5.0f;
 
     public int nfolds = 5;
-    public boolean keep_cross_validation_predictions = true;
-    public boolean keep_cross_validation_models = true;
+    public boolean keep_cross_validation_predictions = false;
+    public boolean keep_cross_validation_models = false;
```

Comment:
@ledell would you agree to align the defaults here with the model properties defaults as defined in Model.java below

Reply:
@ledell this requires documentation changes, so please let me ask confirmation again before I merge this, thanks